core/src/main/java/pl/agh/edu/model/time/Time.java

```diff
@@ -11,8 +11,8 @@
 
 public class Time {
 	private static Time instance;
-	public static final int timeUnitInMinutes = 1;
-	public static final float interval = 5;
+	public static final int timeUnitInMinutes = 60;
```

Comment:
zostało po testowaniu

Reply:
chciałem to zortawić już tak żeby też inne rzeczy było łatwiej testować i tak w zamyśle chceliśmy to prze parametryzować na inne wartości ,ale jak chcesz żeby było tak jak wcześniej to spoko

Reply:
nie no można zostawić, mi to nie przeszkadza